zanata-war/src/main/webapp/WEB-INF/faces-config.xml

```diff
@@ -78,4 +78,387 @@
       <redirect/>
     </navigation-case>
   </navigation-rule>
+
+  <!-- Landing page for open id authentication -->
+  <navigation-rule>
+    <from-view-id>/openid.xhtml</from-view-id>
+    <navigation-case>
+      <if>#{authenticationManager.authenticated and authenticationManager.newUser}</if>
+      <to-view-id>/profile/create_user.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+    <navigation-case>
+      <if>#{!identity.isLoggedIn() and authenticationManager.authenticatedAccountWaitingForActivation}</if>
+      <to-view-id>/account/inactive_account.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+    <navigation-case>
+      <if>#{!identity.loggedIn and applicationConfiguration.singleOpenIdProvider}</if>
+      <to-view-id>/home.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+    <navigation-case>
+      <if>#{identity.loggedIn and userRedirect.shouldRedirectToDashboard()}</if>
+      <to-view-id>/dashboard/home.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+    <navigation-case>
+      <if>#{!identity.loggedIn}</if>
+      <to-view-id>/account/login.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+    <navigation-case>
+      <!-- FIXME [CDI] this will not work if continue url is /project/list, it can not find this as view id (url rewrite)-->
+      <if>#{identity.loggedIn and userRedirect.redirect}</if>
+      <to-view-id>#{userRedirect.url}</to-view-id>
+      <redirect/>
+    </navigation-case>
+    <navigation-case>
+      <!-- Default redirection after a log in is to the dashboard -->
+      <if>#{identity.loggedIn}</if>
+      <to-view-id>/dashboard/home.xhtml</to-view-id>
+      <redirect/>
+    </navigation-case>
+  </navigation-rule>
```

Comment:
On a previous comment I noted that one of the actions (`LoginAction`) had new Java code to determine navigation. Seems like this is the same thing but in `faces-config.xml`. Should we standardize on one place?